django_mongodb/query_utils.py

```diff
@@ -8,7 +11,13 @@ def is_direct_value(node):
 def process_lhs(node, compiler, connection, bare_column_ref=False):
     if not hasattr(node, "lhs"):
         # node is a Func or Expression, possibly with multiple source expressions.
-        return [expr.as_mql(compiler, connection) for expr in node.get_source_expressions()]
+        result = []
+        for expr in node.get_source_expressions():
+            try:
+                result.append(expr.as_mql(compiler, connection))
+            except FullResultSet:
```

Comment:
These lines correspond to https://github.com/django/django/blob/e99187e5c94516ee35f37cc41a36d906b395808d/django/db/models/expressions.py#L1095-L1096.